lib/domain-shared/shared_use_cases/save_image_use_case.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:io';
+
+import 'package:firebase_storage/firebase_storage.dart';
+import 'package:path/path.dart' as path_lib;
+import 'package:seeds/domain-shared/base_use_case.dart';
+import 'package:uuid/uuid.dart';
+
+class SaveImageUseCase extends InputUseCase<String, SaveImageUseCaseInput> {
```

Comment:
this is soo prettty!!!! 😍

Reply:
Do we need the UUID package I think I did a class with that name in other place 🤔